src/test/testScripts/hover/testHoverLocal.test.ts

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable quotes */
+import * as vscode from 'vscode';
+import {
+  getDocUri,
+  activate,
+  testHover,
+  sleep,
+  resetDefaultSettings,
+} from '../../helper';
+
+export function testHoverLocal(): void {
+  describe('TEST FOR HOVER', () => {
+    const docUri1 = getDocUri('hover/1.yml');
+
+    beforeEach(async () => {
+      await resetDefaultSettings();
+      // await vscode.commands.executeCommand('workbench.action.closeAllEditors');
```

Comment:
Any comment on this?

Reply:
I will remove the commented code. Thanks.